stable/chartmuseum/templates/deployment.yaml

```diff
@@ -89,3 +98,18 @@ spec:
       {{- else }}
         emptyDir: {}
       {{- end -}}
+      {{ if .Values.env.open.GOOGLE_SERVICE_ACCOUNT }}
+      - name: {{ include "chartmuseum.fullname" . }}-gcp
+        secret:
+      {{ if .Values.env.open.GOOGLE_CREDENTIALS_JSON }}
```

Comment:
I guess this should be `.Values.env.secret` as if I set up `env.secret.GOOGLE_CREDENTIALS_JSON` in the values this won't be picked.

Reply:
Yes, I agree with @koalalorenzo, I think the GOOGLE_CREDENTIALS_JSON is rooted in `secret` not in `open`. So to be clear, I think this should be changed from `.Values.env.open.GOOGLE_CREDENTIALS_JSON` to `.Values.env.secret.GOOGLE_CREDENTIALS_JSON`